blivet/__init__.py

```diff
@@ -128,6 +128,7 @@ def enable_installer_mode():
     util.program_log_lock = program_log_lock
 
     flags.installer_mode = True
```

Comment:
It's quite confusing to see the flag still being set here with what the commit message says. :) Would it be possible to remove the above line as part of this commit?